libtest/thread.hpp

```diff
@@ -54,10 +54,6 @@ class Mutex
 
```

Comment:
@SpamapS please also refer to thread.hpp content being changed (removing those lines was also crucial to stop make command failing with build process).